fuzz-target/fuzzlib/src/responderlib.rs

```diff
@@ -38,6 +38,7 @@ pub fn rsp_create_info() -> (common::SpdmConfigInfo, common::SpdmProvisionInfo)
         aead_algo: SpdmAeadAlgo::AES_256_GCM,
         req_asym_algo: SpdmReqAsymAlgo::TPM_ALG_RSAPSS_2048,
         key_schedule_algo: SpdmKeyScheduleAlgo::SPDM_KEY_SCHEDULE,
+        secure_spdm_version: [0x11, 0x12],
```

Comment:
We can drop SPDM1.1 for now.  Ref: https://github.com/jyao1/rust-spdm/issues/484

Reply:
I made a mistake, I'm sorry.
please ignore this comment

Reply:
This field should select the versions of SPDM0277.

Reply:
Updated